entity-management-mongo/src/main/java/eu/europeana/entitymanagement/mongo/repository/VocabularyRepository.java

```diff
@@ -18,13 +18,35 @@ public class VocabularyRepository  {
   Datastore datastore;
   
   private static final String ID = "id";
+  
+  private List<Vocabulary> vocabs = new ArrayList<>();
```

Comment:
please rename to europeana Roles and introduce a getEuropeanaRoles method using laizy loading